focus_converter_base/focus_converter/conversion_configs/gcp/1_commitment_discount_type_S001.yaml

```diff
@@ -0,0 +1,7 @@
+plan_name: Unnest credits.id to get CommitmentDiscountType
+conversion_type: unnest
+column: credits.id
```

Comment:
For CommitmentDiscountType, In the credits, we get **type** as well. Should it map to credits.type instead of credits.id ?

Reply:
@spriharani same as above, how do we deal with array values?